internal/dag/cache.go

```diff
@@ -200,6 +214,82 @@ func (kc *KubernetesCache) Insert(obj interface{}) bool {
 	return false
 }
 
+func toV1Ingress(obj *v1beta1.Ingress) *networking_v1.Ingress {
+
+	var convertedTLS []networking_v1.IngressTLS
+	var convertedIngressRules []networking_v1.IngressRule
+	var paths []networking_v1.HTTPIngressPath
+	var convertedDefaultBackend *networking_v1.IngressBackend
+
+	for _, tls := range obj.Spec.TLS {
+		convertedTLS = append(convertedTLS, networking_v1.IngressTLS{
+			Hosts:      tls.Hosts,
+			SecretName: tls.SecretName,
+		})
+	}
+
+	for _, r := range obj.Spec.Rules {
+
+		if r.HTTP != nil {
+			for _, p := range r.HTTP.Paths {
+				paths = append(paths, networking_v1.HTTPIngressPath{
+					Path: p.Path,
+					Backend: networking_v1.IngressBackend{
```

Comment:
should assign `PathType` from the v1beta1 obj as well, though I guess this can wait until we implement #2135

Reply:
setup pathtype just to make the converter complete